cashshop/packets.go

```diff
@@ -134,6 +134,69 @@ func packetCashShopWishList(sns []int32, update bool) mpacket.Packet {
 	return p
 }
 
+func packetCashShopLoadLocker(storage *CashShopStorage) mpacket.Packet {
+	p := mpacket.CreateWithOpcode(opcode.SendChannelCSAction)
+	p.WriteByte(opcode.SendCashShopLoadLockerDone)
+	
+	items := storage.GetAllItems()
+	p.WriteInt16(int16(len(items)))
+	
+	for _, csItem := range items {
+		// LockerItem.Encode structure from OpenMG
+		p.WriteInt64(int64(csItem.dbID))       // CashId
+		p.WriteInt32(0)                        // UserId (0 for account-wide)
```

Comment:
I don't believe this to be accurate. The UserID should be set to the account ID no matter what